_posts/2023-12-20-mvhs-outreach.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  "MVHS 4CSCC Outreach"
+date:   2023-12-20
+categories: event
+author: Franklin Stewart
+---
+![MVHS 4CSCC Outreach](/images/2023-12-20-mvhs-outreach/20231220-MVHS-outreach-00.png)
+During December, Monument Valley High School (Kayenta, AZ) participated and was introduced to the 4CSCC scientific computing curriculum.    Four teachers involved in the 4CSCC outreach dedicated a day to exploring the 4CSCC curriculum.  The sessions were conducted by Franklin Stewart, who introduced environmental sensors, and their uses in detecting and monitoring volatile organic compounds and particulate matter.  Each class was eager to learn about the Raspberry Pi 400 mobile kit’s performance capabilities. 
+In order to engage students in conversations and the curriculum, we begin by discussing facts that are relevant to the students’ lives. Topics range from soldering jewelry to welding, and we highlight their cultural importance but the adverse consequences they can have on the human body.  More information was presented such as proper use and handling when it comes to burning wood stoves.  “There are numerous accounts of Kayenta residents burning trash, these contaminants remain stagnant in the community.  An extreme trigger for various ailments involving the elderly and younger population.”
```

Comment:
@fbstewartm, you have a quote in this paragraph ("There are numerous..."), but it's not attributed to anyone. Can you add an attribution (e.g., _Their teacher shared that "There are numerous..."_), or otherwise clarify.

Reply:
> @fbstewart, you have a quote in this paragraph ("There are numerous..."), but it's not attributed to anyone. Can you add an attribution (e.g., _Their teacher shared that "There are numerous..."_), or otherwise clarify.

Yes, this is coming from me, as an observer and local.  I have removed the quotes.